### ISSUE TITLE
changing the standard request email

### ISSUE THREAD

**Issue by confirmordeny** [mysociety/whatdotheyknow-theme#1238]
When a user begins to make a new request the default text is as follows "Dear [name of public authority]" followed by three blank lines and ends with "Yours faithfully,". I think there is an opportunity to help steer users in the right direction and reduce the time taken to make a polite and valid FOI request. In particular, there is an opportunity to nudge people away from asking questions and towards requesting recorded information. I would suggest the following wording:

"Dear [name of public authority],

Please provide me with [describe the information/documents you would like to receive].

Thank you,"

This is less formal than the current template which is designed to look like a letter but FOI requests don't need to be formal.

**Comment by confirmordeny** [comment]
Here is a link that explains the thinking re displaying emails as letters:
https://github.com/mysociety/alaveteli/issues/167#issuecomment-3813550

**Comment by RichardTaylor** [comment]
Related:

* https://github.com/mysociety/alaveteli/issues/6164

**Comment by confirmordeny** [comment]
I think this could help with accessibility because we would be writing more of the request for people and setting them off on the right track for requesting recorded information.

**Comment by RichardTaylor** [comment]
I wondered if "Yours faithfully" had become archaic, but Google trends doesn't clearly provide support for that hypothesis: 

<img width="1114" alt="Screenshot 2022-10-29 at 19 49 58" src="https://user-images.githubusercontent.com/547695/198848061-c698c321-5655-405e-8c5a-83848a34410c.png">


https://trends.google.com/trends/explore?date=all&geo=GB&q=%22Yours%20faithfully%22

**Comment by RichardTaylor** [comment]
I agree with the aim here. 

I'll develop the draft:

>Dear [name of public authority],
>
>Would you please release [describe the information you are seeking].
>
>Thanks,

From experience we know users don't always understand we're asking them to replace the text in square brackets.  We'll want to catch instances of the text in square brackets being left in as the system seeks to do elsewhere. 

How about another approach of using the subject line. We already ask users to provide: "A one line summary of the information you are requesting". We could take that and put it into the message template. 

Imagine a user set the subject/title to: "Correspondence between the council and company X". 

The message template could then say:

>Dear [name of public authority],
>
>Would you please release correspondence between the council and company X
>
>[give more details such as a time-frame for your request here]
>
>Thanks,

This is certainly something  we should monitor if we do change, and be prepared to rapidly iterate. It's core to the service.

**Comment by confirmordeny** [comment]
Does "Would you please ..." need a question mark?

**Comment by RichardTaylor** [comment]
A need for a question mark is best avoided. 

I thought: "Please release..." sounded a bit too curt. 

My own recent practice has been to write "Could/would you please release..." without a question mark. 

It appears this is grammatically contentious:

https://forum.wordreference.com/threads/question-mark-after-could-you-tell-send-me-etc.1752615/

I like the suggestion in the above linked discussion that no question mark is required if the recipient can't say no!

**Comment by HelenWDTK** [comment]
This issue is being closed due to a lack of discussion or resolution for over 12 months. Should we decide to revisit this issue in the future, it can be reopened.
